docs/FAQs.md

```diff
@@ -0,0 +1,156 @@
+To create a Docker image for a specific branch, use the following command:
+
+`./build_image.sh -b <branch_name>/<commit_id>`
+
+To create a Docker image for a specific branch and specific tag, use the following command:
+
+`./build_image.sh -b <branch_name> -t <tagname:latest>`
+
+
+### What is the difference between image created using Dockerfile and image created using Dockerfile.dev?
+The image created using Dockerfile.dev has Torchserve installed from source where as image created using Dockerfile has Torchserve installed from pyi distribution.  
+
+## API
+Relevant documents
+- [Torchserve Rest API](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md#model-zoo)
+###  What can I use other than *curl* to make requests to Torchserve?
+You can use any tool like Postman, Insomnia or even use a python script to do so. Find sample python script [here](https://github.com/pytorch/serve/blob/master/docs/default_handlers.md#torchserve-default-inference-handlers).
+
+### How can add a custom API to an existing framework?
+You can a custom API using **plugins SDK** available in Torchserve.
+The [Health check API ](https://github.com/pytorch/serve/blob/master/docs/inference_api.md#health-check-api) is an example of a custom API integrated using plugins SDK.
+Refer to Plugins Documentation for more details.
+
+### How can pass multiple images in Inference request call to my model?
+You can provide multiple data in a single inference request to your custom handler as a key-value pair in the `data` object.
+
+Refer [this](https://github.com/pytorch/serve/issues/529#issuecomment-658012913) for more details.
+
+## Handler
+Relevant documents
+- [Default handlers](https://github.com/pytorch/serve/blob/master/docs/model_zoo.md#model-zoo)
+- [Custom Handlers](https://github.com/pytorch/serve/blob/master/docs/custom_service.md#custom-handlers)
+
+###  How do I return an image output for a model?
+You would have to write a custom handler with the post processing to return image.
+Refer [custom  service documentation](https://github.com/pytorch/serve/blob/master/docs/custom_service.md#custom-handlers) for more details.
+
+### How to enhance the default handlers?
+Write a custom handler that extends the default handler and just override the methods to be tuned.
+Refer  [custom  service documentation](https://github.com/pytorch/serve/blob/master/docs/custom_service.md#custom-handlers) for more details.
+
+### How to serve a model with no-code/zero code?
+Yes, you can deploy your model with no-code/ zero code by using builtin default handlers. Refer [default handlers](https://github.com/pytorch/serve/blob/master/docs/default_handlers.md#torchserve-default-inference-handlers) for more details.
+
+### Is it possible to deploy Hugging Face models?
+Yes, you can deploy Hugging Face models using a custom handler.
+Refer [Huggingface_Transformers](https://github.com/pytorch/serve/blob/master/examples/Huggingface_Transformers/README.md) for example. 
```

Comment:
Please be consistent. In some FAQs I see the "Refer..." sentence on a new line and on others it is not.